README.md

```
@@ -171,9 +184,16 @@ npm run start:http
 ```
 
 
-### Using with Claude Desktop
+### Using with MCP Clients
+
+#### Desktop Extension (Recommended)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)